indico/modules/events/abstracts/models/abstracts.py

```diff
@@ -82,46 +89,63 @@ def __table_args__(cls):
             lazy=True
         )
     )
-    #: Data stored in abstract/contribution fields
-    field_values = db.relationship(
-        'AbstractFieldValue',
+    track = db.relationship(
+        'Track',
         lazy=True,
-        cascade='all, delete-orphan',
+        foreign_keys=[track_id],
         backref=db.backref(
-            'abstract',
-            lazy=True
+            'abstracts',
+            lazy='dynamic'
```

Comment:
do we really want dynamic here? i think showing the list of abstracts for a track is not that uncommon and I can already see us changing this later because we want to be able to joinedload it somewhere and then having to change all the places where it's used as a query object to use `with_parent`